app/controllers/hub/users_controller.rb

```diff
@@ -29,7 +29,7 @@ def index
                    @users.search(params[:search])
                  end
       end
-
+      @users = @users.order(suspended_at: :desc)
```

Comment:
TBH it seems like this is a better fix than the show/hide button, since without this line, hiding suspended users on a given page would end up with only a few users visible, and _with_ this line, you don't really need to hide them since they're all tucked away at the end. Maybe something to bring up with PM - ask them to choose between the two?

Reply:
@mrotondo yes I agree! Thanks for bringing this up. After you commented, I brought it up to the PMs that the reordering means that the button isn't that useful anymore but they seemed to want to move ahead with it anyways, indicating there might be a change in the future. It seems to be a feature that hub users have been asking for a long time so maybe they want to see how users interact with it before changing implementation plans 🤷🏻‍♀️